### PR TITLE
SE-2120 Notifications example

### PR DESCRIPTION
**Pull request by lorenzdec**

### CODE REVIEW

File: examples/use-cases/change-management/NotificationsNotebookSandbox.ipynb [hunk 1]
```diff
@@ -0,0 +1,377 @@
+{
```
Comment (steco) [inline examples/use-cases/change-management/NotificationsNotebookSandbox.ipynb:1]: <p>I would us the scenario of "Transactions have been loaded" (and change the subscription_code to "TransactionsLoaded"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/450/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Comment (steco) [inline examples/use-cases/change-management/NotificationsNotebookSandbox.ipynb:1]: <p>Not sure this comment is valid any more</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/450/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Comment (steco) [inline examples/use-cases/change-management/NotificationsNotebookSandbox.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #2.    subscription_scope = 'subscriptionscope'
</pre><p>Put scope first</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/450/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Comment (steco) [inline examples/use-cases/change-management/NotificationsNotebookSandbox.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #3.    scope="email_scope"
</pre><p>Do we need this?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/450/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Comment (steco) [inline examples/use-cases/change-management/NotificationsNotebookSandbox.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #2.    event_filter ='AllocationLoadCompleted'
</pre><p>What does Event_filter mean?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/450/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Comment (steco) [inline examples/use-cases/change-management/NotificationsNotebookSandbox.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #1.    # create email notifications for each of the subscriptions
</pre><p>not "each of"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/450/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Comment (steco) [inline examples/use-cases/change-management/NotificationsNotebookSandbox.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #9.    # API example
</pre><p>"Create a Webhook notification for the subscription"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/450/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Comment (steco) [inline examples/use-cases/change-management/NotificationsNotebookSandbox.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #19.        print(create_email_notifications_response)
</pre><p>I'd remove this</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/450/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Comment (steco) [inline examples/use-cases/change-management/NotificationsNotebookSandbox.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #21.        notifications_api.list_notifications(subscription_scope, subscription_code)
</pre><p>Print this</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/450/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Comment (lorenzdec) [reply examples/use-cases/change-management/NotificationsNotebookSandbox.ipynb:1]: <p>its used to fill out the matching pattern which is a required field - basically causes an alert if a certain message is found</p>

File: examples/use-cases/change-management/NotificationsNotebookSandbox.ipynb [hunk 2]
```diff
@@ -0,0 +1,319 @@
...
```
Comment (steco) [inline examples/use-cases/change-management/NotificationsNotebookSandbox.ipynb:1]: <p>It's slightly confusing that no actual transactions are loaded to LUSID. Can you say:</p><p> </p><p>This notebook runs through a use case for the notification endpoints. The use case is that, following a data load into LUSID (for example daily feed of Transactions) :</p><ul><li>An subscription is created to report the upload event</li><li>An email notification is added to the subscription to report the event to a set of interested parties</li><li>A webhook notification is added to the subscription to continue processing in a downstream system (this is currently disabled)</li><li>The event is manually triggered</li></ul><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/450/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

**Pull request closed without merging**